xil_cloud/docker-compose.yml

```diff
@@ -365,6 +365,18 @@ services:
       - ./secrets:/home/V2X-Hub/secrets
       - ./sensor_configurations:/home/V2X-Hub/sensor_configurations/
 
+  sensorlib-xml-rpc:
+    image: usdotfhwastoldev/sensorlib-xml-rpc:develop
+    networks:
+      carma_sim_net:
+        ipv4_address: 172.2.0.5
+    container_name: sensorlib-xml-rpc
+    restart: always
+    environment:
+      - ENV LOAD_CARLA_EGG=True
+      - ENV CARLA_VERSION=0.9.10
+      - ENV CARLA_EGG_DIR="/home/CarlaCDASimAdapter/"
```

Comment:
I am adding the comment here just to keep track of the progress. When I had the "ENV" here, it was giving me error saying it cannot contain whitespace. The code change I got from that integration testing is here https://github.com/usdot-fhwa-stol/carma-config/tree/feature/sensor-lib-xml-rpc-test. We encountered an issue where the code freezes on config loading stage. And we are currently waiting on its fixes and integration test on the simulation computer.